playground/envs/.env.uat

```
@@ -0,0 +1,2 @@
+VITE_APP_ID="Uat-12345"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)